package.json

```diff
@@ -34,7 +37,8 @@
     "esri-loader": "3.1.0",
     "@fortawesome/fontawesome-svg-core": "1.2.35",
     "@fortawesome/free-solid-svg-icons": "5.15.3",
-    "@fortawesome/react-fontawesome": "0.1.14"
+    "@fortawesome/react-fontawesome": "0.1.14",
+    "@eeacms/volto-clms-utils":"git+http://github.com/eea/volto-clms-utils.git#develop"
```

Comment:
The same as in https://github.com/eea/volto-clms-theme/pull/120#pullrequestreview-813276200 
I think that we should release volto-clms-utils and then depend on it.

Comments @ionlizarazu ?

Reply:
https://github.com/eea/volto-arcgis-block/blob/d9db479c2436a9ab7bbdff1ea758020dce26b20c/package.json#L41

@erral @ionlizarazu Changed. I mark it as solved.